### PR TITLE
Add new member @maria

### PR DESCRIPTION
**Pull request by mariapao**
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

I'm @mariapg on keybase and my Ethereum address is: 0x5fb50ea4af7CE507c4CA5da26dB7a690367137cA
-----BEGIN PGP SIGNATURE-----
Version: Keybase OpenPGP v2.0.80
Comment: https://keybase.io/crypto

wsBcBAABCgAGBQJcSKU6AAoJEDOqjWsMWLw8KqoH/1MoeKOa68sYFm0olDIz9la3
FeavbQsi5MqOz4XyT53p5MKKC1dHy9tULAI/b8CawgS7dQ5QvJroJqgMWTKc96Lr
YtS01l5WPR/ROq7TsJAOhvN9S1Aoi95W6wUJy8BGFBBDXAtqgnSK7nKps9tRsKHM
bHGjVfUUsc3UxpM0W3+fkZlPKJJR2trok4gfc07nEwXsQQH29dgjke/x5xZBNneV
9XiDXJKrvD2b2qw61kxAX0eWGz4Urj7ChHixj7tqpSvlbkgcUUMuqnWnY4ifShe3
vDvVfLhzqhi0jVDVWUXwkkRNxEsiKF6t7u07nhaMTIo+uTlbbHYYRZp5r+44oD0=
=6bWv
-----END PGP SIGNATURE-----

### CODE REVIEW

**Review by lkngtn — APPROVED** [review]
Looks good, Signature checks out. :)

**Comment by maria** [comment]
Hello,

When you use @ on GitHub you are mentioning a username on this platform, so
please make sure you are writing to the intended person.
Please edit your reference to my username, because @maria is me.

Thank you

File: add new member.md [hunk 1]
```diff
@@ -13,4 +13,4 @@
 | <a target="_blank" href="https://keybase.io/schmidsi">@schmidsi</a> | `0xe98C357458C89198575a32462760402CaB2A9a4D` |
 | <a target="_blank" href="https://keybase.io/goliat">@goliat</a> | `0x3E0B02c9f2c5107c2747fCb0E68d997bE2fa0F7f` |
 | <a target="_blank" href="https://keybase.io/dizzypaty">@dizzypaty</a> | `0xa2e9cbaF5Aef1D1781dE1664f50E50FE3481aab7` |
-| | |
+| <a target="_blank" href="https://keybase.io/maria">@maria</a> | `0x5fb50ea4af7CE507c4CA5da26dB7a690367137cA` |
```
Comment (bingen) [inline add new member.md]: Isn't mariapg your keybase username? maria seems to be Maria Kolakowska

**Review by bingen — CHANGES_REQUESTED** [review]

**Comment by lkngtn** [comment]
I think if you edit the signed message so it is code blocks it should not notify others. Though as bingen points out your keybase username is actually mariapg.

**Review by bingen — APPROVED** [review]

**Pull request merged**